src/components/Globals/Row.ts

```diff
@@ -3,4 +3,5 @@ import styled from 'styled-components/native';
 export const Row = styled.View`
   flex-direction: row;
   flex: 1;
+  justify-content: center;
```

Comment:
Metrics aligned left looks more cleaner and evenly distributed to me, let me know your thoughts on this

![1](https://user-images.githubusercontent.com/46997126/81509925-8a35a300-9327-11ea-80fd-0667b787a3f8.PNG)

if so removing justify-content: center; property aligns it left.

Reply:
I'll update this